hw1/02-palindrome.js

```diff
@@ -1,3 +1,58 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput(elem) {
+  ///check for negative input
+  if (elem.target.value < 0) {
+    document.getElementById("result").innerHTML =
+      "That's a negative number. Please enter a positive number.";
+  }
+  //check for single digits
+  else if (elem.target.value < 10) {
+    document.getElementById("result").innerHTML = "Yes, this is a palindrome!";
+    document.getElementById("result").style.color = "green";
+  }
+  //check all positive numbers 2 digits or greater
+  else {
+    let query = elem.target.value;
```

Comment:
I believe these variables `query, text, textLen` don't change through out the program after they are declared.  We could replace them with `const`.  There are more variables through out the program that could be changed to const.  
I'm not sure if there is any effect at all and I don't think it's really recommended by Airbnb but I think it is slightly better and code reader knows what to expect more.